ads/google/ima/ima-video.js

```diff
@@ -950,6 +950,7 @@ export function onContentPauseRequested(global) {
     adsManagerHeightOnLoad = null;
   }
   adsActive = true;
+  playerState = PlayerStates.PLAYING;
```

Comment:
is `playerState` only tracking for ad playing state?

Reply:
Both. The play button is shared, and would toggle to pause based on that state.